mipengine/node/monetdb_interface/merge_tables.py

```diff
@@ -0,0 +1,47 @@
+from typing import List
+
+from mipengine.node.monetdb_interface import common
+from mipengine.node.monetdb_interface import tables
+from mipengine.node.monetdb_interface.common import connection
+from mipengine.node.monetdb_interface.common import convert_schema_to_sql_query_format
+from mipengine.node.monetdb_interface.common import cursor
+from mipengine.node.tasks.data_classes import TableInfo
+from mipengine.utils.custom_exception import IncompatibleSchemasMergeException
+from mipengine.utils.custom_exception import TableCannotBeFound
+
+
+def get_merge_tables_names(context_id: str) -> List[str]:
+    return common.get_tables_names("merge", context_id)
```

Comment:
It is common practice to define a function that calls another function with some specified arguments because if something changes, for example the merge name, you only have to change it in one place, the get_merge_table_names function.